pages/docs/concepts/asyncapi-document/structure.md

```diff
@@ -0,0 +1,432 @@
+
+- `schemas`: An object to hold reusable Schema Object. 
+- `servers`: An object to hold reusable Server Objects.
+- `channels`: An object to hold reusable Channel Objects.
+- `operations`: An object to hold reusable Operation Item Objects.
+- `messages`: An object to hold reusable Message Objects.
+- `securitySchemes`: An object to hold reusable Security Scheme Objects.
+- `serverVariables`: An object to hold reusable Server Variable Objects.
+- `parameters`: Contains reusable parameter objects that can be used in various parts of the AsyncAPI document.
+- `correlationIds`: An object to hold reusable Correlation ID Objects.
+- `replies`: An object to hold reusable Operation Reply Objects.
+- `replyAddresses`: An object to hold reusable Operation Reply Address Objects.
+- `externalDocs`: An object to hold reusable External Documentation Objects.
+- `tags`: An object to hold reusable Tag Objects.
+- `operationTraits`: An object to hold reusable Operation Trait Objects.
+- `messageTraits`: Represents common traits or characteristics that can be applied to messages.
+- `serverBindings`: An object to hold reusable Server Bindings Objects.
+- `channelBindings`: An object to hold reusable Channel Bindings Objects.
+- `operationBindings`: An object to hold reusable Operation Bindings Objects.
+- `messageBindings`: An object to hold reusable Message Bindings Objects.
+
+Here's a visual representation of the `components` object and its properties:
+```mermaid
+graph LR
+  A[components]
+  B(schemas)
+  C(Category)
+  D(Tag)
+  G(servers)
+  H(development)
+  I(serverVariables)
+  L(channels)
+  P(messages)
+  U(parameters)
+  W(correlationIds)
+  Y(messageTraits)
+
+  A --> B
+  B --> C
+  B --> D
+  A --> G
+  G --> H
+  H --> I
+  A --> L
+  A --> P
+  A --> U
+  A --> W
+  A --> Y
+```
+
+Here's a code example of the components object in an AsyncAPI document:
+```yaml
+components:
+  schemas:
+    Category:
+      type: object
+      properties:
+        id:
+          type: integer
+          format: int64
+        name:
+          type: string
+    Tag:
+      type: object
+      properties:
+        id:
+          type: integer
+          format: int64
+        name:
+          type: string
+    AvroExample:
+      schemaFormat: application/vnd.apache.avro+json;version=1.9.0
+      schema:
+        $ref: 'path/to/user-create.avsc/#UserCreate'
+  servers:
+    development:
+      url: "{stage}.in.mycompany.com:{port}"
+      description: RabbitMQ broker
+      protocol: amqp
+      protocolVersion: 0-9-1
+      variables:
+        stage:
+          $ref: "#/components/serverVariables/stage"
+        port:
+          $ref: "#/components/serverVariables/port"
+  serverVariables:
+    stage:
+      default: demo
+      description: This value is assigned by the service provider, in this example `mycompany.com`
+    port:
+      enum: [5671, 5672]
+      default: 5672
+  channels:
+    user/signedup:
+      subscribe:
```

Comment:
This is not a valid channel object in v3. You can use the one you described [here](https://github.com/asyncapi/website/pull/1922/files#diff-a8927db92ee3b3aeae4648a24b55a4091178117ac9ad24466491db1377c4fe15R175)

Reply:
Just to double check I got it correct, are you suggested I use the channel object described above as follows👇🏻?

Current:
"channels:
    user/signedup:
      subscribe:"

Suggested:
"channels:
     userSignedUp:
          subscribe:
          message:
          $ref: '#/components/messages/userSignedUp'